pkg/project/controller/controller_test.go

```diff
@@ -38,35 +38,63 @@ func TestSyncNamespaceThatIsTerminating(t *testing.T) {
 		t.Errorf("Unexpected error when handling namespace %v", err)
 	}
 
-	// TODO: we will expect a finalize namespace call after rebase
+	namespaceFinalize := core.NewCreateAction(unversioned.GroupVersionResource{Group: "", Version: "", Resource: "namespaces"}, "", testNamespace)
+	namespaceFinalize.Subresource = "finalize"
+
 	expectedActionSet := []core.Action{
-		core.NewListAction(unversioned.GroupVersionResource{Group: "", Version: "v1", Resource: "buildconfigs"}, "", kapi.ListOptions{}),
-		core.NewListAction(unversioned.GroupVersionResource{Group: "", Version: "v1", Resource: "policies"}, "", kapi.ListOptions{}),
-		core.NewListAction(unversioned.GroupVersionResource{Group: "", Version: "v1", Resource: "imagestreams"}, "", kapi.ListOptions{}),
-		core.NewListAction(unversioned.GroupVersionResource{Group: "", Version: "v1", Resource: "policybindings"}, "", kapi.ListOptions{}),
-		core.NewListAction(unversioned.GroupVersionResource{Group: "", Version: "v1", Resource: "rolebindings"}, "", kapi.ListOptions{}),
-		core.NewListAction(unversioned.GroupVersionResource{Group: "", Version: "v1", Resource: "roles"}, "", kapi.ListOptions{}),
-		core.NewListAction(unversioned.GroupVersionResource{Group: "", Version: "v1", Resource: "routes"}, "", kapi.ListOptions{}),
-		core.NewListAction(unversioned.GroupVersionResource{Group: "", Version: "v1", Resource: "templates"}, "", kapi.ListOptions{}),
-		core.NewListAction(unversioned.GroupVersionResource{Group: "", Version: "v1", Resource: "builds"}, "", kapi.ListOptions{}),
-		core.NewListAction(unversioned.GroupVersionResource{Group: "", Version: "v1", Resource: "namespace"}, "", kapi.ListOptions{}),
-		core.NewListAction(unversioned.GroupVersionResource{Group: "", Version: "v1", Resource: "deploymentconfig"}, "", kapi.ListOptions{}),
-		core.NewListAction(unversioned.GroupVersionResource{Group: "", Version: "v1", Resource: "egressnetworkpolicy"}, "", kapi.ListOptions{}),
+		namespaceFinalize,
+		core.NewListAction(unversioned.GroupVersionResource{Group: "", Version: "", Resource: "buildconfigs"}, testNamespace.Name, kapi.ListOptions{}),
```

Comment:
> Yes, at the bare minimum you had to copy paste one of the NewListAction lines.

For code that we know we're going to delete, that has a fixed point where we're going to delete it, that we know we're going to backport, I'm not sure I'd make the larger change.

@mfojtik have you reasoned through the test to see if it still works?  I haven't, but if we really want to go to the effort and we know this change is ok then actually checking is clearly better.